etc/config-template.toml

```diff
@@ -785,6 +785,15 @@
 # enable-statistics = true
 # stats-dump-period = "10m"
 
+## RocksDB WAL directory.
+## This config specifies the absolute directory path for WAL.
+## If it is not set, the log files will be in the same directory as data. When you set the path to
+## RocksDB directory in memory like in `/dev/shm`, you may want to set`wal-dir` to a directory on a
+## persistent storage. See https://github.com/facebook/rocksdb/wiki/How-to-persist-in-memory-RocksDB-database .
+## If there are two disks on the machine, storing RocksDB data and WAL logs on different disks can
+## improve performance.
+# wal-dir = ""
```

Comment:
```suggestion
## This config specifies the absolute directory path for WAL.
## If it is not set, the log files will be in the same directory as data. 
## If there are two disks on the machine, storing RocksDB data and WAL logs on different disks can
## improve performance.
# wal-dir = ""
```
We don't recommend putting the WAL in tempfs.

Reply:
> We don't recommend putting the WAL in tempfs.

It seems that the original comment means the same with you. It suggest that if the user set the rocksdb directory in tempfs, the user is supposed to set this config to a persistent storage.

Reply:
Indeed I misread the comment. But anyway we shouldn't mention tempfs. Both WAL and the SST files should be on persistent disk.

Reply:
I agree with you. I've deleted it in the latest commit.